types: ClientCapabilityOptions makes ttl required but it's optional.

### Issue Summary
The typescript definition of `ClientCapabilityOptions` [makes ttl required](https://github.com/twilio/twilio-node/blob/0018eca40e914d9149b49a3e6b198f4704dc82b2/lib/jwt/ClientCapability.d.ts#L47-L51), but the [code implies a default of 3600](https://github.com/twilio/twilio-node/blob/0018eca40e914d9149b49a3e6b198f4704dc82b2/lib/jwt/ClientCapability.js#L115)

### Steps to Reproduce
In a typescript project, use twilio@latest. Attempt to instantiate `jwt.ClientCapability` without ttl and notice there's a typescript error.

### Code Snippet
```node
import { jwt } from 'twilio';

const client = new jwt.ClientCapability({  
  accountSid: '',
  authToken: ''
})
```

I'd suggest updating the definition: 
```
  export interface ClientCapabilityOptions {
    accountSid: string;
    authToken: string;
    ttl?: number;
  }
```

### Exception/Log
```
error TS2345: Argument of type '{ accountSid: string; authToken: string; }' is not assignable to parameter of type 'ClientCapabilityOptions'.
  Property 'ttl' is missing in type '{ accountSid: string; authToken: string; }' but required in type 'ClientCapabilityOptions'.
```

### Technical details:
* twilio-node version: 3.55.0
* node version: 10.19.0 (but would happen on any)

--- a/lib/jwt/ClientCapability.d.ts
+++ b/lib/jwt/ClientCapability.d.ts
@@ -47,7 +47,7 @@ declare namespace ClientCapability {
   export interface ClientCapabilityOptions {
     accountSid: string;
     authToken: string;
-    ttl: number;
+    ttl?: number;
   }
 }
 

